app/src/routes/routesMap.js

```diff
@@ -78,6 +79,8 @@ export default {
   API_PAGE: '/api',
 
   [PROJECTS_PAGE]: '/administrate/projects',
+  [PROJECT_DETAILS_PAGE]:
+    '/administrate/project-details/:projectId/:projectSection(settings|members|events)?',
```

Comment:
We discussed this route with Ilya, and agreed on:
`/administrate/:projectId/:projectSection(settings|members|events)?`

Reply:
Changed